src/views/TradingCompetition/components/TeamRanks/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Flex, Box, Image } from '@pancakeswap-libs/uikit'
+import CakerBunny from '../../pngs/cakers.png'
+import TopTradersCard from './TopTradersCard'
+import Podium from './Podium'
+
+const Wrapper = styled(Flex)`
+  flex-direction: column;
+
+  ${({ theme }) => theme.mediaQueries.md} {
+    flex-direction: row;
+  }
+`
+
+const StyledPodiumWrapper = styled(Flex)`
+  margin: 0 0 40px 0;
```

Comment:
Unless you want to explicitly set  the top, right, left margin I think `margin-bottom: 40px;` is better.

Reply:
👍